### PR TITLE
Icom 705 support

### PR DESCRIPTION
**Pull request by mckoss**
First pass - I was able to download to my Icom 705.  Still need to test "on the air" to make sure the settings are correct.

I reformatted the markdown file to 100 columns where possible - hope that's ok with you.

### CODE REVIEW

File: files.md [hunk 1]
```diff
@@ -9,28 +9,62 @@ In order to make the contesting life a little easier, we provide some files you
...
+## Icom-705 Instructions
+
+*These have been tested with the Icom-705.  If you have an Icom 7300 (or other Icom radio), please
+let us know if this file is also compatible with your radio.*
+
+1. Download and install the [Icom CS-705
+   Software](https://www.icomjapan.com/support/firmware_driver/3067/).
+2. Connect a USB cable from your computer to the Icom 705.
+3. Use the `COM Port > Setting...` command to select the COM port used by the USB connection.
+4. Use the `Clone < Read <- TR` menu to duplicate the Icom's settings to your computer.
```
Comment (QCaudron) [inline files.md:66]: What is the `<` notation, compared with the `>` ? And the `<-` ?

File: scripts/update.py [hunk 2]
```diff
@@ -375,6 +376,66 @@ def format_df_for_d878(df: pd.DataFrame) -> pd.DataFrame:
...
+    -------
+    pd.DataFrame
+        The Icom-compativle repeaters formatted for the Icom.
+        (Tested with Icom-705 - may work for others).
+    """
+
+    total_repeaters = len(df)
+
+    # Select FM and Fusion (in FM compat mode) channels.
+    df = df.loc[df["Mode"].isin(["FM", "NBFM", "Fusion"])].copy()
```
Comment (QCaudron) [inline scripts/update.py:398]: The 705 can do DSTAR too, right ? Shall we add those later ?

**Comment by mckoss** [comment]
It's not notation - that's literally what is in the menu text.

- Mike
(425) 246-7701 (m)

**Comment by mckoss** [comment]
Yes! Are there dstar channels in the database?  I actually didn't notice
them.

- Mike
(425) 246-7701 (m)

File: scripts/update.py (hunk 2 above)
Comment (mckoss) [reply scripts/update.py:398]: Yes! Are there dstar channels in the database?  I actually didn't notice
them.

File: files.md (hunk 1 above)
Comment (mckoss) [reply files.md:66]: It's not a notation - it's the literal text of the menu.

    Read <- TR

Comment (mckoss) [reply files.md:66]: Oh - but I did get the `>` backwards!

File: scripts/update.py (hunk 2 above)
Comment (QCaudron) [reply scripts/update.py:398]: There are a handful. They don't take PL tones or anything specific, so assuming the existing code can deal with nulls or `""` under Tone, we should get them for free by including them on this line !

Comment (mckoss) [reply scripts/update.py:398]: I'd like to add this support in a followup PR, if that's OK.

**Review by QCaudron — APPROVED** [review]

**Pull request merged**